packages/graphql-server-lambda/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "declaration": true,
+    "noImplicitAny": false,
+    "rootDir": "./src",
+    "outDir": "./dist",
+    "allowSyntheticDefaultImports": false,
+    "pretty": true,
+    "removeComments": true,
+    "typeRoots": [
+        "node_modules/@types"
+    ],
+    "types": [
+        "@types/graphql",
```

Comment:
are you sure you need @types/graphql in here?
it think is requested as dep of core so you get's it from there..